BUILDER_DEV.md

```diff
@@ -1,221 +1,67 @@
-
-### Pre-Reqs
-
-* Ubuntu 17.10 Desktop or Ubuntu 17.10 Server Virtual Machine
-* Access to the "core" origin in [Production Builder](https://bldr.habitat.sh). Ask a core maintainer (or in the [Habitat Slack](http://slack.habitat.sh/)) if you do not have access.
-* Access to the Habitat 1Password account (NOTE - we are working on a solution to make this not required, but it is necessary for the time being)
-* All of the following steps should be run within your Virtual Machine
-* A [Github Personal Access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) with all repo and all user permissions.
-* The [Habitat Builder App](https://github.com/apps/habitat-builder) installed on your Github account.
-* Remove the "secure_path" section from your sudoers file (this is required for the make scripts to run correctly). The following command will comment it out:
-```
-$ sudo sed -i.bak '/secure_path/s/^/#/' /etc/sudoers
-```
-* Some of the sample commands below use the 'httpie' tool. Install it if not present on your system (https://github.com/jkbrzt/httpie).
-
-
-### Builder Setup
-* Update your system
-```
-$ sudo apt-get update
-```
-
-* NOTE - if you are running Ubuntu 17.10 Server, the above command may fail with "E: The repository 'cdrom://Ubuntu-Server 17.10 _Artful Aardvark_ - Release amd64 (20171017.1) artful Release' does not have a Release file."  To fix this, run:
-
-```
-$ sudo sed -i.bak '/deb cdrom/s/^/#/g' /etc/apt/sources.list
-```
-
-* Install git
-```
-$ sudo apt-get install git
-```
-
-* Clone the [Habitat repo](https://github.com/habitat-sh/habitat)
-```
-$ git clone https://github.com/habitat-sh/habitat.git
-```
-
-* Copy habitat-builder-dev.2017-10-02.private-key.pem from 1Password into path/to/habitat/repo/.secrets/builder-github-app.pem (NOTE - this is required as of 12/11/17 - we are working on a solution to make access to the Habitat 1Password vault not required for dev environment setup).
-* Set up some environmental variables (I put these in my .bashrc)
-
-```
-export HAB_AUTH_TOKEN="<your github token>"
-export HAB_BLDR_URL="http://localhost:9636"
-export HAB_ORIGIN="core"
-```
-
-* symlink /src to path/to/your/habitat/repo
-
-```
-$ sudo ln -s path/to/your/habitat/repo /src
-```
+## Pre-Requisites
 
-* Cd into your Habitat repo
+### GitHub OAuth Application
 
-```
-$ cd /src
-```
+1. Create a GitHub Organization
+1. [Setup a GitHub application](https://github.com/settings/apps/new) for your GitHub organization.
+1. Set the value of `Homepage URL` to `http://${APP_HOSTNAME}`
+1. Set the value of `User authorization callback URL` to `http://${APP_HOSTNAME}/` (The trailing `/` is *important*)
+1. Set the value of `Webhook URL` to `http://${APP_HOSTNAME}/`
+1. Set everything to read only (this is only used for your org so it's safe)
+1. Save and download the pem key
```

Comment:
For folks who don't already know, what `pem` is, it might be clearer to say "generate private key" and that it will result in a file like `app-name.date.private-key.pem`.

Reply:
updated